src/core_modules/capture-core/components/WidgetEnrollment/Actions/Transfer/hooks/useUpdateOwnership.js

```diff
@@ -46,11 +46,12 @@ export const useUpdateOwnership = ({
     onAccessLostFromTransfer,
 }: Props): ReturnTypes => {
     const dataEngine = useDataEngine();
-    const teiParamKey = useFeature(FEATURES.newTrackedEntityQueryParam) ? 'trackedEntity' : 'trackedEntityInstance';
     const { show: showErrorAlert } = useAlert(
         i18n.t('An error occurred while transferring ownership'),
         { critical: true },
     );
+    const teiParamKey = useFeature(FEATURES.newTrackedEntityQueryParam) ? 'trackedEntity' : 'trackedEntityInstance';
+    const orgUnitKey = useFeature(FEATURES.orgUnitReplaceOuQueryParam) ? 'orgUnit' : 'ou';
```

Comment:
The Cypress test `User can transfer the enrollment to another organisation unit` is still [failing](https://github.com/dhis2/capture-app/actions/runs/14733270154/job/41478280091?pr=4063). I think using the new orgUnit query param when building the URL in [@with-transfer-ownership-data-cleanup](https://github.com/dhis2/capture-app/blob/master/cypress/e2e/WidgetsForEnrollmentPages/WidgetEnrollment/index.js#L7) will fix the scenario.

Reply:
Thanks! Missed this one.